Karma/Models/ReportDataModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Karma.Models
+{
+    public class ReportDataModel
+    {
+        public List<Report> Reports { get; set; }
+        public int? page { get; set; }
+        public string sorting { get; set; }
+        public string? aggr { get; set; }
```

Comment:
`string aggr ` can be set to null just like `string? aggr`. What's intended here?

Reply:
Thanks for spotting mistake. Fixed.